app/scripts/utils/colours.es6

```diff
@@ -10,3 +10,35 @@ export default {
   poor: '#f05e55',
   unknown: '#7d7d7d',
 };
+
+export const Color = {};
+
+Color.getWaterpointColor = (status) => {
+  let color = null;
+  switch (status) {
+  case 'FUNCTIONAL':
+    color = 'green';
```

Comment:
Should we match these colours to the theme's red/gred/yellow? We could get the colour here from the map above, like `colours.good` instead of `green` (if we name the exported object "colours").